app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
@ethanresnick I prefer `automatic` because it tells me as someone using the API that I am not supposed to worry about this property (same with `internal`). For me, `independent` could just as easily mean that the system does not manage this property and so I have to make sure I do the bookkeeping for it.